ReactWindows/ReactNative.Shared/Views/Control/ReactControl.cs

```diff
@@ -33,30 +34,30 @@ public ReactControl()
         /// <summary>
         /// The view children.
         /// </summary>
-        public UIElementCollection Children
-        {
-            get
-            {
-                return _canvas.Children;
-            }
-        }
+        public UIElementCollection Children => _canvas.Children;
 
         /// <summary>
         /// Keys that should be handled during <see cref="UIElement.KeyDownEvent"/>. 
         /// </summary>
-        public int[] HandledKeyDownKeys
-        {
-            get;
-            set;
-        }
+        public int[] HandledKeyDownKeys { get; set; }
 
         /// <summary>
         /// Keys that should be handled during <see cref="UIElement.KeyUpEvent"/>. 
         /// </summary>
-        public int[] HandledKeyUpKeys
+        public int[] HandledKeyUpKeys { get; set; }
+
+        /// <inheritdoc />
+        protected override AutomationPeer OnCreateAutomationPeer()
         {
-            get;
-            set;
+            return new AccessibleAutomationPeer<ReactControl>(this);
         }
+
+        // TODO: implement runtime change raising event to screen reader #1228350
```

Comment:
>1228350 [](start = 74, length = 7)

As mentioned elsewhere, let's reference GitHub issue numbers rather than Skype issue numbes

Reply:
good catch, changed to a github issue #

---
In reply to: [158562334](https://github.com/reseul/react-native-windows/pull/1#discussion_r158562334) [](ancestors = 158562334)